README.md

```diff
@@ -13,7 +13,7 @@ There is also a collection of accessibility-related utility code, including but
 
 To include just the javascript rules, require the following file:
 
-    https://raw.github.com/GoogleChrome/accessibility-developer-tools/stable/dist/js/axs_testing.js
+    https://rawgit.com/GoogleChrome/accessibility-developer-tools/stable/dist/js/axs_testing.js
```

Comment:
Good catch on pointing to the production URL. According to [rawgit.com](http://rawgit.com) though, the URL we should use is https://cdn.rawgit.com/GoogleChrome/accessibility-developer-tools/master/dist/js/axs_testing.js

Could you please update this PR with that change?